db/schema.rb

```diff
@@ -271,6 +266,14 @@
 
   add_index "content_buckets_homerooms", ["content_bucket_id", "homeroom_id"], name: "content_bucket_homeroom_index", unique: true, using: :btree
 
+  create_table "content_buckets_purchases", id: false, force: true do |t|
```

Comment:
wait a minute....this came from your first commit on this branch: https://github.com/calblueprint/worldreader/commit/c944fa936256f6647213687aabfc25bf8c355532